internal/pkg/service/buffer/api/service/service.go

```diff
@@ -58,28 +65,118 @@ func (*service) CreateReceiver(d dependencies.ForProjectRequest, payload *buffer
 	// Generate Secret
 	receiver.Secret = idgenerator.ReceiverSecret()
 
+	for _, exportData := range payload.Exports {
+		export := model.Export{
+			Name: exportData.Name,
+			ImportConditions: model.ImportConditions{
+				Count: 1,
+				Size:  100,
+				Time:  30 * time.Second,
+			},
+		}
+
+		if exportData.Conditions != nil {
+			export.ImportConditions.Count = exportData.Conditions.Count
+			export.ImportConditions.Size = datasize.ByteSize(exportData.Conditions.Size)
+			export.ImportConditions.Time = time.Duration(exportData.Conditions.Time) * time.Second
```

Comment:
Tu sme asi chceli tiez parsovat string nie? `10h` je lepsie pre uzivatela, ako pisat to v sekundach, `time.ParseDuration() `.